slides-03-design.qmd

```diff
@@ -0,0 +1,364 @@
+
+> For analysis of the complete set of cases using pre-election data only, including all
+cases and representative of the 2020 electorate, use the full sample pre-election
+weight, **V200010a**. For analysis including post-election data for the complete set of
+participants (i.e., analysis of post-election data only or a combination of pre- and
+post-election data), use the full sample post-election weight, **V200010b**. Additional
+weights are provided for analysis of subsets of the data, as follows.
+
+For weight | Use variance unit/PSU/cluster | and use variance stratum
+-----------|-------------------------------|-------------------------
+V200010a| V200010c| V200010d
+V200010b| V200010c| V200010d
+
+
+## Example: ANES 2020 Syntax {.smaller}
+
+```{r}
+#| label: design-anes-example
+anes <- anes_2020 %>%
+  mutate(Weight = V200010b / sum(V200010b) * 231592693) # <1>
+
+anes_des <- anes %>%
+  as_survey_design(
+    weights = Weight, #<2>
+    strata = V200010d, #<3>
+    ids = V200010c, #<4>
+    nest = TRUE
+  ) #<5>
+
+summary(anes_des) #<6>
+```
+
+1. Adjust the weight of the ANES data to reflect the national population
+2. Specify the weight variable
+3. Specify the strata variable per documentation
+4. Specify the cluster variable per documentation
+5. Indicate that the clusters are nested within strata
+6. Summarize the design object
+
+## National Health and Nutrition Examination Survey (NHANES) {.smaller}
+
+- Analysis weight: WTINT2YR
+- Variance Stratum: SDMVSTRA
+- Variance Primary Sampling Unit: VPSU
+- Large population
+
+::: {.panel-tabset}
+
+### Fill in the blank
+
+```r
+nhanes_des <- nhanes %>%
+   as_survey_design(
+      weights = ___________,
+      ids = ___________, 
+      strata = ___________, 
+      fpc = ___________, 
+   )
+```
+
+### Solution
+
+```r
+nhanes_des <- nhanes %>%
+   as_survey_design(
+      weights = WTINT2YR,
+      ids = VPSU,
+      strata = SDMVSTRA,
+      fpc = NULL
+   )
+```
+
+:::
+
+# Replicate weight methods {background-color='{{< brand color secondary >}}'}
+
+## Replicate weights overview {.smaller}
+
+Replicate weights are another method to estimate variability. In general, they are contstructed by:
+
+1. Divide the sample into subsample replicates that mirror the design of the sample
+2. Calculate weights for each replicate using the same procedures for the full-sample weight (i.e., nonresponse and post-stratification)
+3. Calculate estimates for each replicate using the same method as the full-sample estimate
+4. Calculate the estimated variance, which is proportional to the variance of the replicate estimates
+
+## Common replicate weight methods
+
+- Balanced repeated replication (BRR)
+- Fay's BRR
+- Jackknife
+- Bootstrap
+
+## Specify the survey object with replicate weights
+
+
+- [This](http://gdfe.co/srvyr/reference/as_survey_rep.html) creates a `tbl_svy` object that then correctly calculates weighted estimates and SEs
+
+```{r}
```

Comment:
This code runs off the slides.  I've shorted up the type text below to make it fit better.